keymaps/darwin.cson

```diff
@@ -73,7 +73,8 @@
   'cmd-alt-right': 'pane:show-next-item'
   'ctrl-pageup': 'pane:show-previous-item'
   'ctrl-pagedown': 'pane:show-next-item'
-  'ctrl-tab': 'pane:show-next-item'
+  'ctrl-tab': 'pane:show-next-recently-used-item'
+  'ctrl-tab ^ctrl': 'pane:move-item-to-top-of-stack'
```

Comment:
Maybe this should be called `pane:move-active-item-to-top-of-stack`. I dunno. @nathansobo you are the naming master.